cmd/dockerd/config_unix.go

```diff
@@ -58,7 +58,7 @@ func installConfigFlags(conf *config.Config, flags *pflag.FlagSet) error {
 	flags.StringVar(&conf.InitPath, "init-path", "", "Path to the docker-init binary")
 	flags.Int64Var(&conf.CPURealtimePeriod, "cpu-rt-period", 0, "Limit the CPU real-time period in microseconds for the parent cgroup for all containers")
 	flags.Int64Var(&conf.CPURealtimeRuntime, "cpu-rt-runtime", 0, "Limit the CPU real-time runtime in microseconds for the parent cgroup for all containers")
-	flags.StringVar(&conf.SeccompProfile, "seccomp-profile", "", "Path to seccomp profile")
+	flags.StringVar(&conf.SeccompProfile, "seccomp-profile", "", `Path to seccomp profile. Use "unconfined" to disable the default seccomp profile`)
```

Comment:
Should this default to `config.SeccompProfileDefault` so that the effective default value shows up in the help output?

Reply:
I was a bit in doubt; I think for most options we consider `""` (default string value) to also be `default`. The ambiguity was a bit in that when using `docker info`, it will show `default` (if nothing is set), so yes, from that perspective perhaps it makes sense to use it here as well. 😅